tests/x11regressions/evolution/evolution_meeting_pop.pm

```diff
@@ -0,0 +1,351 @@
+    if (match_has_tag 'evolution-default-client-ask') {
+        assert_and_click "evolution-default-client-agree";
+        assert_screen "test-evolution-1";
+    }
+    send_key "$next";
+    assert_screen "evolution_wizard-restore-backup";
+    send_key "$next";
+    assert_screen "evolution_wizard-identity";
+    wait_screen_change {
+        send_key "alt-e";
+    };
+    type_string "SUSE Test";
+    wait_screen_change {
+        send_key "alt-a";
+    };
+    type_string "$mail_box";
+    send_key "$next";
+    assert_screen "evolution_wizard-skip-lookup";
+    assert_screen "evolution_wizard-account-summary";
+
+    #if used Yahoo account, need disabled Yahoo calendar and tasks
+    if ($i eq "Yahoo") {
+        send_key "alt-l";
+    }
+    send_key "$next";
+    if (sle_version_at_least('12-SP2')) {
+        send_key "$next";    #only in 12-SP2 or later
+        send_key "ret";
+    }
+    assert_screen "evolution_wizard-done";
+    send_key "alt-a";
+    if (check_screen "evolution_mail-auth") {
+        if (sle_version_at_least('12-SP2')) {
+            send_key "alt-a";    #disable keyring option, only in SP2 or later
+            send_key "alt-p";
+        }
+        type_string "$mail_passwd";
+        send_key "ret";
+    }
+    if (check_screen "evolution_mail-init-window") {
+        send_key "super-up";
+    }
+    assert_screen "evolution_mail-max-window";
+}
+
+sub send_meeting_requst {
+
+    my ($self, $sender, $receiver, $mail_subject) = @_;
+    my $config      = $self->getconfig_emailaccount;
+    my $mail_box    = $config->{$receiver}->{mailbox};
+    my $mail_passwd = $config->{$sender}->{passwd};
+
+    #create new meeting
+    send_key "shift-ctrl-e";
+    assert_screen "evolution_mail-compse_meeting", 30;
+    send_key "alt-a";
+    sleep 2;
+    type_string "$mail_box";
+    send_key "alt-s";
+    if (sle_version_at_least('12-SP2')) {
+        send_key "alt-s";    #only need in sp2
+    }
+    type_string "$mail_subject this is a pop test meeting";
+    send_key "alt-l";
+    type_string "the location of this meetinng is conference room";
+    assert_screen "evolution_mail-compse_meeting", 60;
+    send_key "ctrl-s";
+    assert_screen "evolution_mail-sendinvite_meeting", 60;
+    send_key "ret";
+    if (check_screen "evolution_mail-auth") {
+        if (sle_version_at_least('12-SP2')) {
+            send_key "alt-a";    #disable keyring option, only need in SP2 or later
+            send_key "alt-p";
+        }
+        type_string "$mail_passwd";
+        send_key "ret";
+    }
+    assert_screen "evolution_mail-compse_meeting", 60;
+    send_key "ctrl-w";
+    if (check_screen "evolution_mail-save_meeting_dialog") {
+        send_key "ret";
+    }
+    if (check_screen "evolution_mail-send_meeting_dialog") {
+        send_key "ret";
+    }
+    if (check_screen "evolution_mail-meeting_error_handle") {
+        send_key "alt-t";
+    }
+}
+
+sub check_new_mail {
```

Comment:
Hi coolo,  thanks for your comments.
This function is a little different with the one in evolution_meeting_imap.pm. POP3 protocol won't keep the mail in the server, so it is no need to delete the new mail. It is different with  IMAP and other mail protocol.
I working on reducing the cost time of imap meeting case,  could I put this function to x11regressiontest when I update imap meeting case？